File: Capítulo 04 - Pilhas/teste.js
let lista = [1,2,3,4,5];

function pesquisaIndice(lista, numero) {
    let local = lista.indexOf(numero);
    if (numero in lista) {
        console.log(`\nArray: [${lista}]\nPosição: ${local + 1}\nIndice: ${local}`);
    }
    else {
        console.log('Número Inexistente');
    }
}

// pesquisaIndice(lista, 20);

const soma = (array) => {
    let resultado = 0;
    for (let valores of array) {
        resultado += valores;
    }
    return console.log(resultado);
}

let media = soma(lista) / lista.length;
console.log(media);

File: Capitulo 04 - Pilhas/teste.py
lista = [1,2,3,4,5]

def pesquisaIndice(lista, numero):
    if numero in lista:
        print('Número Existe')
    else:
        print('Número Inexistente')

# pesquisaIndice(lista, 20)

def media(*valores):
    lista = list(valores)
    media = sum(lista) / len(lista)
    return print(media)

media(4,4,4)